pynestml/meta_model/ast_neuron.py

```diff
@@ -325,11 +324,10 @@ def get_parameter_symbols(self):
                 ret.append(symbol)
         return ret
 
-    def get_state_symbols(self):
+    def get_state_symbols(self) -> Iterable[VariableSymbol]:
```

Comment:
Wouldn't it better to have `List` here instead of `Iterable`?

Reply:
You're right, this is probably better as it's more specific. Having an Iterable instead of a List might be desirable as an input parameter type, but is too nonspecific as a return type.